docs/lua-modules/liquidcrystal.md

```diff
@@ -237,6 +237,12 @@ liquidcrystal:blink(true)
 ## liquidcrystal.busy
 Get busy status of the LCD. When using GPIO backend without `rw` argument specification function does nothing.
 
+!!! note
+  At least some HD44780s and/or interfaces have been observed to count polling
+  the busy flag as grounds for incrementing their position in memory.  This is
```

Comment:
```suggestion
  At least some HD44780s and/or interfaces have been observed to increase their cursor position
  when polling the busy flag.  This is
```

Not entirely sure but I think this better expresses what is happening. I think instead of 'ground' you mean 'reason'? but again my English knowledge might not be complete here.

Reply:
Your English is excellent and certainly better than my anything-not-English.  "ground" is, tragically, one of those very, very heavily overloaded terms; I was using sense 4 of https://www.merriam-webster.com/dictionary/ground , but I think you're right that it's wordy and your phrasing is better. :)

Reply:
Just out of curiosity. What is your mother tongue then since you seem to live in GB? Mine is German.

Reply:
Tragically I am one of those people from the USA who speak only English and, worse, I decided to study Latin for my foreign language requirements growing up. XD  Your English is *definitely* better than my German.